cpp/src/node_options.cpp

```diff
@@ -19,15 +19,17 @@
  *
  */
 
-#include "proton/codec/vector.hpp"
```

Comment:
Not sure why this has been removed. It doesn't seem related to your changes. If it's not needed it would be better (and less distracting) to remove it in a separate change (it could be NO-JIRA if you're really sure it's not needed - effectively that is what this change is!)